drivers/bluetooth/hci/Kconfig

```diff
@@ -131,3 +131,18 @@ config BT_DRIVER_QUIRK_NO_AUTO_DLE
 
 	  This has to be enabled when the BLE controller connected is Zephyr
 	  open source controller.
+
+
+config BT_HCI_H4_SETUP
+	bool "Enable HCI vendor-specific Setup function"
+	depends on BT_H4
+	help
+	  Enable the HCI vendor-specific Setup function.
+
+	  This option has to be enabled when the BT Controller requires execution
+	  of the vendor-specific commands sequence to initialize the BT Controller
+	  before the BT Host executes a Reset sequence.
+
+	  The user should generally avoid changing it via menuconfig or in
+	  configuration files. This option are enabled by the vendor-specific
+	  HCI extension, where the Setup function is implemented.
```

Comment:
Not sure I understand this paragraph. The only way to enable it seem to be from menuconfig or config files.

Reply:
hi  @Thalley.
user will select BT device  ( e.g BT_CYW43XXX, and it  selects BT_HCI_H4_SETUP)

```
config BT_CYW43XXX
	bool "CYW43XXX BT connectivity"
	select GPIO
	select BT_HCI_H4_SETUP
```
NOTE: support of BT_CYW43XXX will added in another MR.

Reply:
Okay, that explains it. 

Is it OK for the user to `select` manually? If not, then you can remove the text after `bool` to make it a hidden Kconfig, that can only be enabled by e.g. `BT_CYW43XXX`.

Reply:
For me, it makes sense to hide BT_HCI_H4_SETUP.

Reply:
Then I suggest we do that :)

Reply:
done.